src/firebase/CRUDServices/imageService.js

```diff
@@ -1,12 +1,12 @@
 import firebase from '../firebase';
+import 'firebase/storage';
 
 export default class ImageService {
   constructor() {
-    self.storageRef = firebase.storage().ref();
-    self.sieRawImagesBucketRef = storageRef.child('sie-raw-images');
-    self.sieProcessedImagesBucketRef = storageRef.child('sie-stimuli-images');
-    self.topicName =
-    'projects/cs6510-spr2021/topics/sie-image-processing-result-test';
+    self.db = firebase.firestore();
+    self.images = 'Images';
```

Comment:
`images` is a collection right? Can we rename this appropriately?

Reply:
umm, maybe imagesCollection?  I was trying to be consistent with the other two services(user/experiment) but can be changed to make it more readable.